### ISSUE TITLE
I don't see the heart rate graph after import of a .Fit file from Suunto

### ISSUE THREAD

**Issue by torebo** [thomaschampagne/elevate#734]
### Step 1: Are you in the right place?

  * Your problem may have been already covered in the FAQ. Did you check https://github.com/thomaschampagne/elevate/wiki/Frequently-Asked-Questions? yes
  * Your problem or feature request may already exists in project issues section. Did you check https://github.com/thomaschampagne/elevate/issues? yes

### Step 2: Describe your environment

  * Plugin version: _____
  * Chrome version: Version 71.0.3578.98 (Build officiel) (64 bits)
  * OS version: Windows 10 pro version 1803
  
### Step 3: Describe the problem:
I uploaded to Strava the .fit file attached in zip file, that was generated on movescount website. This is a Badminton activity
[Move_2019_01_07_12_26_54_Badminton.zip](https://github.com/thomaschampagne/elevate/files/2736139/Move_2019_01_07_12_26_54_Badminton.zip)
when I look at the result on my Strava profile https://www.strava.com/activities/2061955062, It seems something wrong happens: the HR data graph is missing
My Strava account: 

#### Steps to reproduce:

  1. _____go to https://www.strava.com/activities/2061955062
  2. _____look at the graph in overview
  3. _____
  
#### Actual Behavior:

  * What happened?
  * Do you have javascript errors logs in browser console? NO

  > Press F12 to see developer console and javascript logs

  ```javascript
  Copy and paste your console logs here.
  ```jsapi?key=ABQIAAAA9S76sdh5KSPpfr65zxQHGBT-rU0CFRjazioGWeeHjJLEyYXO2RSmaFbEdGganPrQRY2l3ORmYrTPpA:23 A parser-blocking, cross site (i.e. different eTLD+1) script, https://www.google.com/uds/?file=gdata&v=1.x, is invoked via document.write. The network request for this script MAY be blocked by the browser in this or a future page load due to poor network connectivity. If blocked in this page load, it will be confirmed in a subsequent console message. See https://www.chromestatus.com/feature/5718547946799104 for more details.
google.loader.f @ jsapi?key=ABQIAAAA9S76sdh5KSPpfr65zxQHGBT-rU0CFRjazioGWeeHjJLEyYXO2RSmaFbEdGganPrQRY2l3ORmYrTPpA:23
P.load @ jsapi?key=ABQIAAAA9S76sdh5KSPpfr65zxQHGBT-rU0CFRjazioGWeeHjJLEyYXO2RSmaFbEdGganPrQRY2l3ORmYrTPpA:31
google.load @ jsapi?key=ABQIAAAA9S76sdh5KSPpfr65zxQHGBT-rU0CFRjazioGWeeHjJLEyYXO2RSmaFbEdGganPrQRY2l3ORmYrTPpA:18
(anonymous) @ 2061955062:3139
2jsapi?key=ABQIAAAA9S76sdh5KSPpfr65zxQHGBT-rU0CFRjazioGWeeHjJLEyYXO2RSmaFbEdGganPrQRY2l3ORmYrTPpA:23 A parser-blocking, cross site (i.e. different eTLD+1) script, https://www.google.com/uds/api/gdata/1.10/208a907929e7936990d2bf48205aa8d8/core,mediarss,geo,opensearch,atom,threading,app,gdata,finance,contacts,analytics,blogger,maps,gbase,acl,calendar,default.I.js, is invoked via document.write. The network request for this script MAY be blocked by the browser in this or a future page load due to poor network connectivity. If blocked in this page load, it will be confirmed in a subsequent console message. See https://www.chromestatus.com/feature/5718547946799104 for more details.
google.loader.f @ jsapi?key=ABQIAAAA9S76sdh5KSPpfr65zxQHGBT-rU0CFRjazioGWeeHjJLEyYXO2RSmaFbEdGganPrQRY2l3ORmYrTPpA:23
(anonymous) @ ?file=gdata&v=1.x:8
boot.bundle.js:73 A previous sync exists on Tue Jan 08 2019 10:40:17 GMT+0100 (heure normale d’Europe centrale)
boot.bundle.js:73 Fast sync mode enabled
boot.bundle.js:73 Local and remote activities count matches.
boot.bundle.js:73 Last sync date time saved:  Tue Jan 08 2019 11:18:30 GMT+0100 (heure normale d’Europe centrale)
boot.bundle.js:73 Fast sync finished {activitiesChangesModel: {…}, syncedActivities: Array(154), lastSyncDateTime: 1546942710289}
  
#### Expected Behavior:

  * What did you expect to happen?
  I expected the HR data graph to be shown

#### Media attachement:

>You can upload/attach any images to this ticket.
strava: https://i.gyazo.com/4b701c70b98c2c92a55215f16f40e246.png
same activity on Movescount: https://i.gyazo.com/0f7c58cc8f13f80028ade54dd601546b.png

#### Providing your Elevate synced athlete activities can be usefull to solve the problem

> To export your activities click on "sync button" (top right and corner), then "Backup activities". You can zip the file and upload it here.

**Comment by thomaschampagne** [comment]
@torebo You should flag your activity as "Run" to get heart rate data.

Because support of generic activities is not supported yet. Linking https://github.com/thomaschampagne/elevate/issues/617

I close this ticket since https://github.com/thomaschampagne/elevate/issues/617 will handle it
